db/external_sst_file_ingestion_job.cc

```diff
@@ -248,14 +248,16 @@ Status ExternalSstFileIngestionJob::Run() {
     // is written to the database.
     uint64_t oldest_ancester_time = 0;
     int64_t temp_current_time = 0;
+    uint64_t current_time = static_cast<uint64_t>(temp_current_time);
     if (env_->GetCurrentTime(&temp_current_time).ok()) {
```

Comment:
This needs to be fixed; copy/paste error I believe. You first need to set `temp_current_time` by doing `env_->GetCurrentTime(&temp_current_time)` before casting it to uint64_t.